woo-razorpay.php

```diff
@@ -315,9 +287,22 @@ public function autoEnableWebhook()
 
             $key_id      = $this->getSetting('key_id');
             $key_secret  = $this->getSetting('key_secret');
-            $enabled     = $this->getSetting('enable_webhook');
-            $secret      = $this->getSetting('webhook_secret');
+            $enabled     = 'yes';
+            $secret      =  bin2hex(openssl_random_pseudo_bytes(4));
```

Comment:
@jampulanaveen Can we use bugger length secrets here also if it must have capital, small case letters and numbers.